### PR TITLE
Belligerent now prevents you from running even between chants

### LINKED ISSUE #26511: broken <span in ratvar code
There's some missing HTML syntax here
`<span class='userdanger'You feel something massive turn its crushing focus to you...`

Server revision compiled on: 2017-04-23
The following pull requests are currently test merged:
#26465: 'Fixes Advanced Proc Call' by Cyberboss 

Based off master commit:
164f7f9b5431ac0e32987313fc896fed46a952bd 

````
[Common] Krill Newsman says, "Like clockwork"
"You will do, heretic."
<span class='userdanger'You feel something massive turn its crushing focus to you...
[Common] Jemmy Stange says, "A shoutout to the cob scarab that took care of the AI for me"
"No matter. I will find you later, heretic."
You feel tremendous relief as the crushing focus relents...
[Common] Jemmy Stange says, "You are a true hero"
"You will do, heretic."
<span class='userdanger'You feel something massive turn its crushing focus to you...
"No matter. I will find you later, heretic."
You feel tremendous relief as the crushing focus relents...
````

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Belligerent now prevents you from running for 7 seconds after every chant.
rscdel: However, you no longer remain walking after Belligerent's effect fades.
/:cl:

Fixes #26511

### CODE REVIEW

**Comment by Cyberboss** [comment]
FREEEZE MADAME!!

File: code/datums/status_effects/debuffs.dm [hunk 1]
```diff
@@ -29,3 +29,50 @@
...
+				owner.apply_damage(leg_damage_on_toggle * 0.5, BURN, "l_leg")
+				owner.apply_damage(leg_damage_on_toggle * 0.5, BURN, "r_leg")
+		if(owner.m_intent != MOVE_INTENT_WALK)
+			if(!iscultist(owner))
+				to_chat(owner, "<span class='warning'>Your leg[number_legs > 1 ? "s shiver":" shivers"] with pain!</span>")
+			else //Cultists take extra burn damage
+				to_chat(owner, "<span class='warning'>Your leg[number_legs > 1 ? "s burn":" burns"] with pain!</span>")
+			owner.toggle_move_intent()
+		return TRUE
+	return FALSE
```
Comment (coiax) [inline code/datums/status_effects/debuffs.dm]: Is there a click delay on attempt to toggle running, or will people be able to knock themselves into crit by spam clicking.

File: code/game/gamemodes/clock_cult/clock_structures/ratvar_the_clockwork_justicar.dm [hunk 2]
```diff
@@ -76,7 +76,7 @@
 		if(!prey && LAZYLEN(meals))
 			prey = pick(meals)
 			to_chat(prey, "<span class='heavy_brass'><font size=5>\"You will do, heretic.\"</font></span>\n\
-			<span class='userdanger'You feel something massive turn its crushing focus to you...</span>")
+			<span class='userdanger'>You feel something massive turn its crushing focus to you...</span>")
```
Comment (coiax) [inline code/game/gamemodes/clock_cult/clock_structures/ratvar_the_clockwork_justicar.dm]: A T O M I Z E

File: code/datums/status_effects/buffs.dm [hunk 3]
```diff
@@ -13,6 +13,7 @@
 /datum/status_effect/shadow_mend/on_apply()
 	owner.visible_message("<span class='notice'>Violet light wraps around [owner]'s body!</span>", "<span class='notice'>Violet light wraps around your body!</span>")
 	playsound(owner, 'sound/magic/Teleport_app.ogg', 50, 1)
+	return ..()
```
Comment (coiax) [inline code/datums/status_effects/buffs.dm]: Shouldn't this be `. = ..()` for speed?

File: code/datums/status_effects/debuffs.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/datums/status_effects/debuffs.dm]: they can yes, but it takes several seconds even hammering the button so you'd have to do it intentionally and if you do I don't give a shit

**Comment by ShadowDeath6** [comment]
quick!

File: code/datums/status_effects/buffs.dm (hunk 3 above)
Comment (Cyberboss) [inline code/datums/status_effects/buffs.dm]: literally no difference as they boil down to the same opcode

Comment (coiax) [inline code/datums/status_effects/buffs.dm]: @RemieRichards has told me different, that apparently the `return` keyword is somehow shitter than just the functioning ending.

**Comment by KorPhaeron** [comment]
PRs that violated the freeze have to wait a week longer to be PRd or else people will keep opening their feature PRs during freezes

File: code/datums/status_effects/buffs.dm (hunk 3 above)
Comment (duncathan) [inline code/datums/status_effects/buffs.dm]: actually it was profiled and iirc determined that `.` is only faster than return for modifying the return and then letting it implicitly pass; `return` should be faster here

**Pull request merged**